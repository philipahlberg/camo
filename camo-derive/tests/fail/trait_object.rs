use camo_derive::Camo;

#[derive(Camo)]
struct Foo {
    x: dyn std::fmt::Debug,
}

fn main() {}
