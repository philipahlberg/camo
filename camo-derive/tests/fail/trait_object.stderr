error: `camo` does not support this type
 --> tests/fail/trait_object.rs:5:8
  |
5 |     x: dyn std::fmt::Debug,
  |        ^^^
